core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -1021,7 +1030,7 @@ class AuthorizerIntegrationTest extends AbstractAuthorizerIntegrationTest {
    * The newer version is covered by testAuthorizationWithTopicNotExisting.
    */
   @ParameterizedTest
-  @CsvSource(value = Array("false", "true"))
+  @ValueSource(booleans = Array(true, false))
```

Comment:
Should we change the ordering in `testTopicIdAuthorization` too to be consistent??